strawberryfields/api_client.py

```diff
@@ -154,6 +150,7 @@ def __init__(self, **kwargs):
         Initialize the API client with various parameters.
         """
         self._config = self.get_configuration_from_config()
+        self._config["debug"] = self._config["debug"] in (True, "True", "true", "TRUE", 1)
```

Comment:
How come the casting of `self._config["debug"]` happens here? Perhaps this check should go in the `Configuration` class, so that it is applied to all boolean configuration options from hereon

Reply:
@josh146 Good idea. I did initially put this in `Configuration` but it seemed a little overkill -- however I can see how this would be helpful in the future for any boolean configuration option. Will change.

Reply:
Fixed.